plugins/svelte/deps.ts

```diff
@@ -0,0 +1,11 @@
+export * as path from "https://deno.land/std@0.86.0/path/mod.ts";
+
+export {
+  compile,
+  preprocess,
+} from "https://raw.githubusercontent.com/crewdevio/Snel/main/compiler/core.js";
```

Comment:
It would be preferable to use a fixed import version so that consumers don’t find sub deps change under their feet and potentially break. For instance from https://deno.land/x/snel@v0.0.5

Reply:
done